ts/components/Pinpad/index.tsx

```diff
@@ -24,13 +25,15 @@ interface Props {
 
 interface State {
   value: string;
+  disabled: boolean;
```

Comment:
we usually prepend boolean variable names with `is`:

```suggestion
  isDisabled: boolean;
```